rachellegelden/Home-Together-Canada
# HTC-131: Static Home Page

# [Issue 131](https://github.com/rachellegelden/Home-Together-Canada/issues/131)
Closes #131 

## Summary
- Added Tailwindcss CSS framework to project
- Updated `.gitignore` to ignore `Tailwindcss.output.css` file
- Updated ESLinter settings to send `warnings` for `prop-types` and `unknown-react` instead of `error`
- Created `images` folder and added temp HTC logo  
- Created Header for unregistered users
- Created Footer for unregistered users
- Created basic home page layout (refer to wireframe)

## Relevant Motivation & Context
- Updated `.gitignore` to ignore `Tailwindcss.output.css` file = because `Tailwindcss.output.css` changes based on the page and is auto-recoded by Tailwindcss when the local server starts
- Updated ESLinter settings to send `warnings` for `prop-types` and `unknown-react` instead of `error`= because @Lightbrite88 was getting some errors on his end and this update ensures those errors are not blocking code compiling

## Testing Instructions
- Header = double-check to ensure `html` tags are accurate 
- Footer = double-check to ensure `html` tags are accurate 

## Developer checklist prior to opening this pull request:


## Reviewer
